front/src/api/user.ts

```diff
@@ -1,11 +1,84 @@
 import GigzFetcher from '../services/GigzFetcher';
-import IGigzResponse from '../types/IGigzResponse';
 import EProfileType from '../types/EProfileType';
 import IArtistProfile from '../types/IArtistProfile';
+import IGenre from '../types/IGenre';
+import IGigzResponse from '../types/IGigzResponse';
 import IHostProfile from '../types/IHostProfile';
 
+interface IProfilePicture {
+  media: string;
+}
+
+export interface IGalleryPhoto {
+  media: string;
+  id: number;
+}
+
 export const getProfile = async (
   profileType: EProfileType
 ): Promise<IGigzResponse<IArtistProfile | IHostProfile>> => {
   return GigzFetcher.get(`me/${profileType}`);
```

Comment:
Why do we specify the porfile type here ? Isn't the back supposed to know from the token what profile this is